src/FirstSteps/RPG/Heroes/Dwarf.cs

```diff
@@ -13,6 +14,20 @@ public Dwarf(string name) : base(name, Races.Dwarf)
             _health = 25;
             _damage = 7;
         }
+        public Dwarf(string name, Races race, int strength, int intelligence, int agility, int health, int damage, int coinsBag) : base(name, Races.Dwarf)
+        {
+            _heroModel = new HeroModel
```

Comment:
Here you should assign values from parameters to private variables. Just like in the constructor above but here you have params instead constant values. 

All comments from `Dwarf.cs` are also applicable to the rest of the heroes.